proposed/phpdoc.md

```diff
@@ -253,8 +242,8 @@ format, as dictated by the specific tag.
 > _type_ (`string`), variable name (`$argument1`),  and description (`This is a
 > parameter.`).
 
-The description MUST support Markdown as a formatting language.  The
-description of the tag MAY start on the same line or next line.  The following
+The description MUST support Markdown as a formatting language. The
+description of the tag MAY start on the same line or next line. The following
```

Comment:
Does this implicitly mean supporting HTML, too?

Reply:
Also... which Markdown? :-)  There are many flavors.

Reply:
Maybe we just have to pick CommonMark as that's the most common standard I know in markdown. The set is limited, but as far as I know, most tools support the syntax of CommonMark. 

I need to do some more research on how other php based tools are doing this, but mostly it's just phpDocumentor doing this. I didn't find any other libraries that handle the description of docblocks in the wild. Apart from the IDE's. We would need some people from those sides to have a look at this.

Reply:
https://github.com/yiisoft/yii2-apidoc